### PR TITLE
Create Active record model

### PR DESCRIPTION
**Pull request by ioilmio**
This creates model for user comment and post;
Add basic validations for those resources.

### CODE REVIEW

**Comment by uryelah** [comment]
Hi **Salvo Illuminato**, I'm Sarah your reviewer for this PR 👩‍🚀 

Good job with this project, you are just missing one relation between the user and comment. Check the required changes below and good coding! 💪 

# Required Changes 🏗️ 🐛 💡

- [ ] A user can write more than one comment and a comment can only have one user author, right? Make sure that you associate the comment and user model accordingly, who belongs to who and who has many of the other?

-----

 **Kindly fix the pointed out issues before requesting the next code review**

------------

Cheers and Happy coding!👏👏👏

Code review by **Sarah Uryelah Silva Chamorro**

(Feel free to contact me on slack by the handle of  **sarah** if you have any additional question)

File: app/models/comment.rb [hunk 1]
```diff
@@ -2,4 +2,5 @@ class Comment < ApplicationRecord
   validates :body, presence: true, length: { in: 2..20 }
 
   belongs_to :posts
+  belongs_to :user_id
```
Comment (uryelah) [inline app/models/comment.rb]: - [x] It shoud belong to :user (as in the User model), not user_id.

**Review by uryelah — CHANGES_REQUESTED** [review]
Hi Salvo, you are almost there. 
Just make sure to follow the rails conventions, I left I comment in your code with the issue you need to fix.

# Required Changes 🏗️ 🐛 💡

File: app/models/comment.rb (hunk 1 above)
Comment (uryelah) [reply app/models/comment.rb]: - [x] Also, if it belongs to one it should be :post, in the singular.

Comment (uryelah) [reply app/models/comment.rb]: https://guides.rubyonrails.org/association_basics.html#the-belongs-to-association

**Review by rOluochKe — APPROVED** [review]
Hello Team!
### Well done, you got it right this time :clap: The project has been approved. Congratulations :boom: :100:
### Merge the project and submit it. Good luck with the next one!

- Happy coding!

**Pull request merged**